src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -295,6 +306,58 @@ class boss_gormok : public CreatureScript
         }
 };
 
+class SnobolledTargetSelector : public std::unary_function<Unit*, bool>
+{
+public:
+    SnobolledTargetSelector(Unit const* unit, float dist = 200.0f) : _me(unit), _dist(dist) { }
+
+    bool operator()(Unit* unit) const
+    {
+        if (unit->GetTypeId() != TYPEID_PLAYER)
+            return false;
+
+        if (_dist > 0.0f && !_me->IsWithinCombatRange(unit, _dist))
```

Comment:
if someone put a custom value for _dist != 200.0f and this value is < 0.0f (is a negative value), convert this negative value in positive again